Minor changes to problem description text. (#261)
workshopper/stream-adventure: problems/meet_pipe/problem.md

A stream is an abstract interface for working with streaming data in Node.js.

> That means you can consume data as it is loaded or produced, chunk by chunk (or
> piece by piece), instead of getting it all into memory in one go.

Reviewer: I think your change is not totally accurate. The difference when you don't use streams is that you have to wait to get all data into memory before you can use it.

I think could be better something like:

```suggestion
That means you can consume data as it is loaded or produced, chunk by chunk (or
piece by piece), instead of getting it all into memory to start consuming it.
```

Author: Ah very good point! Sounds much better
